### DISCUSSION TITLE
Any Examples for Tracing Requests in Microservice Architecture with Jaeger as Backend?

### DISCUSSION THREAD

**Discussion by Nathan-Bernardo** [open-telemetry/community#1307] [Community Discussions]
Hi guys,

I'm new to the whole observability concept and came across Jaeger and OpenTelemetry.  I'm able to implement simple tracers in NestJS, but my goal now is to trace requests that goes from Service A to Service B.  I'm a bit stuck in how I can ensure spans from different services are linked to the same trace.  If anyone has any example or idea on how I can do this, that would be great!  Implementation can be in JS or TS as well to translate the idea.

**Comment by trask** [comment]
hi @Nathan-Bernardo! I'd recommend opening a discussion in https://github.com/open-telemetry/opentelemetry-js (and there's also an [#otel-js slack channel](https://cloud-native.slack.com/archives/C01NL1GRPQR), and [instructions for joining CNCF Slack](https://github.com/open-telemetry/community#discussions) if needed)
